model/Licensing/Classes/AttributionText.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AttributionText
+
+## Summary
+
+Class representing attribution text.
+
+## Description
+
+An AttributionText is used to indicate the attribution text of a software artifact.
+
+## Metadata
+
+- name: AttributionText
+- SubclassOf: xsd:string
```

Comment:
Classes cannot be subclassed from datatypes

Reply:
Yes, @sbarnum ; in RDF it won't be a subclass, but a derived datatype.